packages/lit-dev-content/site/_includes/default.html

```diff
@@ -8,15 +8,22 @@
     <meta name="description" content="Simple. Fast. Web Components.">
     <link rel="icon" href="{{ site.baseurl }}/images/flame-favicon.svg">
     <link rel="mask-icon" href="{{ site.baseurl }}/images/flame-favicon.svg" color="#324fff">
-    <link rel="modulepreload" href="{{ site.baseurl }}/js/global/lit-hydrate-support.js">
+    <script type="modulepreload" src="{{ site.baseurl }}/js/tslib.js"></script>
+    <link rel="modulepreload" href="{{ site.baseurl }}/js/lit.js">
+    <link rel="modulepreload" href="{{ site.baseurl }}/js/global/hydrate-common-components.js">
 
     {% inlinecss "global.css" %}
 
     {% inlinecss "mods.css" %}
     {% inlinejs "global/apply-mods.js" %}
     {% inlinejs "global/initialize-typescript-attribute.js" %}
     {% inlinejs "global/mobile-drawer.js" %}
-    <script type="module" src="{{ site.baseurl }}/js/global/lit-hydrate-support.js"></script>
+    {% if env.DEV %}
+      <script type="module" src="{{ site.baseurl }}/js/global/lit-hydrate-support.js"></script>
+    {% else %}
+      <!-- Loads lit-hydration since we manually chunk this -->
+      <link rel="module" href="{{ site.baseurl }}/js/lit.js">
```

Comment:
A bit confused about this. Should this be `modulepreload`? I couldn't find `module` in the link types: https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types

Reply:
no, because in the rollup config we manually chunk everything in `node_modules/lit` into `lit.js` that means that the files loaded by hydrate helpers are located at the top of this `lit.js` file. If i were to `<script type="module" src="/js/global/lit-hydrate-support.js">` then even in prod it would make 2 render-blocking requests to first `/js/global/lit-hydrate-support.js` then to `/js/lit.js`. We want to stop this from happening so we just load `lit.js` directly

Reply:
Makes sense. I think I'm just confused what `link` with `rel="module"` means?

Reply:
OMG that is a TYPO!

Reply:
how does any of this even work???

Reply:
ohh it's because it's in lit.js and anything loading lit ever will setup correctly. Maybe we don't even need an else statement here

Reply:
Keeping it in here just to be safe